Sources/SwiftDocC/Servers/FileServer.swift

```diff
@@ -56,49 +54,49 @@ public class FileServer {
     /**
      Returns the data for a given URL.
      */
-    public func data(for url: URL) -> Data? {
+    public func data(for path: String) -> Data? {
```

Comment:
This is a source breaking change. Would it be possible to keep the existing function and have it call into the new function?
```suggestion
    @available(*, deprecated, message: "Use 'data(for path: String)' instead.")
    public func data(for url: URL) -> Data? {
        return data(for url.path)
    }

    public func data(for path: String) -> Data? {
```